api/app/signals/apps/email_integrations/reporter/mail_actions.py

```diff
@@ -4,41 +4,33 @@
 from django.conf import settings
 from django.core.mail import send_mail as django_send_mail
 from django.db.models import Q
-from django.template import loader
+from django.template import Context, Template, loader
 from django.utils.text import slugify
 
+from signals.apps.email_integrations.models import EmailTemplate
+from signals.apps.signals import workflow
 from signals.apps.signals.models import Signal
-from signals.apps.signals.workflow import (
-    AFGEHANDELD,
-    AFWACHTING,
-    BEHANDELING,
-    GEANNULEERD,
-    GEMELD,
-    GESPLITST,
-    HEROPEND,
-    INGEPLAND,
-    ON_HOLD,
-    VERZOEK_TOT_AFHANDELING,
-    VERZOEK_TOT_HEROPENEN
-)
 
 SIGNAL_MAIL_RULES = [
     {
         'name': 'Send mail signal created',
         'conditions': {
             'filters': {
-                'status__state__in': [GEMELD, ],
+                'status__state__in': [workflow.GEMELD, ],
```

Comment:
Just a nitpick. Not blocking.

Why use `workflow.` instead of the way it was implemented?
I like to only import what I need to use just to keep the amount of imports as low as possible.

Reply:
In my option this makes the import part much more readable.

Reply:
This is not a blocker for me. So resolved.